app/src/main/res/values/strings.xml

```diff
@@ -16,7 +16,11 @@
     <string name="condition3">" and "</string>
     <string name="condition4">"Privacy Policy"</string>
     <string name="signGoogle">"Sign in with Google"</string>
+
+    <string name="graphScreenTitle">Calories Per Day</string>
+
```

Comment:
extra space?

Reply:
this came from merging didn't pay attention